src/components/Progress.tsx

```diff
@@ -1,10 +1,18 @@
 import 'react-sweet-progress/lib/style.css';
 
 import React from 'react';
+// react-sweet-progress의 타입을 지정하는 package가 없는것 같아서 repo를 찾아보니 ts-ignore로 처리하라는 답변을 보고 작성해둠.
+// @ts-ignore
```

Comment:
```suggestion
// @ts-ignore
```

https://github.com/nickbullll/react-sweet-progress/issues/14

그렇네요. 이건 우선 이렇게 해결하고 추후에 [chakra ui](https://chakra-ui.com/docs/components/progress/usage) 마이그레이션하면서 변경하면 될 것 같아요! 좋습니다

Reply:
이슈를 멘션했다고 흔적이 남는군요?
새로운거 알아갑니다!